guide/C/ch_invest.xml

```diff
@@ -1817,6 +1817,1129 @@ Income
+        reducing manual calculation especially if dividends have been
+        reinvested over years and there are many different costs involved
+        in a sale.
+      </para>
+
+      <para>The term <emphasis>scrub</emphasis> is used because security
+        accounts need to be cleaned up after sales to ensure the 
+        difference between the cost paid for securities, and value
+        received from selling them, is accounted for as capital gain or
+        loss. If the capital gain/loss is not correct, the Trial Balance
+         (<menuchoice>
+            <guimenu>Reports</guimenu>
+            <guisubmenu>Income &amp; Expense</guisubmenu>
+            <guimenuitem>Trial Balance</guimenuitem>
+          </menuchoice>) bottom line total debits will not balance to 
+          total credits.
+      </para>
+                    
+      <sect3 id="invest-sellLimitations">
+        <title>Limitations</title>
+        
+        <para>There are some limitations to using this feature.</para>
+        
+        <itemizedlist>
+          <listitem>
+            <para><application>&app;</application> has only 1 method of
+              automatically linking buy transactions to sell transactions -
+              First In/ First Out (FIFO). I.e. The oldest securities are always
+              sold first.
+            </para>
+          </listitem>
+          
+          <listitem>
+            <para>The Advanced Portfolio Report does not use lot information
+              when calculating costs. It calculates the cost basis and gains
+              or losses based on the selected <emphasis>Basis calculation method
+              </emphasis> report option (Average, FIFO or LIFO). I.e. it does
+              not use any of the lot information, just the security transaction
+              splits.
+            </para>
+          
+            <para>If you are selling <emphasis>all</emphasis> of your holding of
+              a particular security, it doesn't matter which cost method you
+              use as the total cost of the securities sold will be the same.
+              It is only when you have partially sold a security, that the 
+              costing method is important, so that the Advanced Portfolio Report
+              costs (and therefore gains or losses) are correct for the 
+              remainder of the security holding going forward. As the costing
+              method is a report option, it is preferable that the same costing
+              method is used for all securities. Otherwise, it may be necessary
+              to run multiple Advanced Portfolio Reports, 1 for each costing
+              method and only including the securites using that costing method.
+            </para>
+            
+            <note>
+              <para>Provision has been made in <application>&app;</application>
+                for specifying costing method per security account but this is
+                yet to be implemented.
```

Comment:
I think you're explaining the "scrub lots" function here, but you need to be clear about it; it appears the note applies to the Advance Portfolio Report.